FusionIIIT/applications/central_mess/handlers.py

```diff
@@ -610,7 +610,7 @@ def generate_bill():
                                        month=previous_month,
                                        year = previous_month_year,
                                        total_bill=total):
-            print("exists")
+           1
```

Comment:
Is this condition required? [exists()](https://docs.djangoproject.com/en/4.1/ref/models/querysets/#django.db.models.query.QuerySet.exists) can be used to check if an object exists.

Reply:
Although it won't matter to pass the condition as we have to check if the object is already exist and then do nothing, used the exists() function in the if condition now in the latest commit.

Reply:
You can remove that if condition since its not used to make any changes.